modules/gapi/samples/face_detection.cpp

```diff
@@ -454,6 +461,7 @@ GAPI_OCV_KERNEL(OCVTranspose, Transpose) {
 namespace vis {
 namespace {
 void bbox(const cv::Mat& m, const cv::Rect& rc) {
+    std::cout << "Final rectangle " << "x1 = " << rc.x << " y1 = " << rc.y << " x2 = " << rc.x + rc.width << " y2 = " << rc.y + rc.height << std::endl;
```

Comment:
Is it debug or log string? It is per frame spam.

Reply:
Done